pkg/data/mongo.go

```diff
@@ -2,12 +2,16 @@ package data
 
 import (
 	"context"
+	"time"
 
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.mongodb.org/mongo-driver/mongo/readpref"
 )
 
+// CONNECTIONTIMEOUT is a constant to set the waiting time for a database connection
+const CONNECTIONTIMEOUT = 60 * time.Second
```

Comment:
Earlier I suggested you naming it in upper case, sorry. In golang, it'd be expected to use `ConnectionTimeout` or `connectionTimeout`.

```suggestion
const connectionTimeout = 60 * time.Second
```

Reply:
Done!